miner/src/pool/scoring.rs

```diff
@@ -108,6 +204,16 @@ impl<P> txpool::Scoring<P> for NonceAndGasPrice where P: ScoredTransaction + txp
 					super::Priority::Regular => 0,
 				};
 				scores[i] = scores[i] << boost;
+				let last_index = txs.len() - 1;
+				for idx in 1..last_index {
+					scores[idx - 1] = *txs[idx-1].transaction.gas_price()
+						+ is_consecutive(&txs[idx-1..=idx])
+						+ (U256::from(21_000) / scores[idx - 1])
```

Comment:
I suppose this should be `gas` not `score`?

Reply:
Yeahp, just didn't add gas() to `ScoredTransaction` yet, so used score as placeholder